common/types/metrics.ts

```diff
@@ -13,10 +19,18 @@ export interface MetricType extends VisualizationType {
   w: number;
   h: number;
   query: {
-    type: 'savedCustomMetric' | 'prometheusMetric';
+    type: 'savedCustomMetric' | 'prometheusMetric' | 'openTelemetryMetric';
```

Comment:
Can re-use the constant declared above: `OTEL_METRIC_SUBTYPE`

Reply:
why is this `prometheusMetric` not `promqlmetric`?

Reply:
reused it. @ps48 
that change was made by Peter's previous PR @joshuali925 i didn't modify it